rts/motoko-rts/src/persistence.rs

```diff
@@ -127,6 +127,17 @@ unsafe fn use_enhanced_orthogonal_persistence() -> bool {
     }
 }
 
+/// Returns the availability of the stable actor record (false on (re-)install, true on upgrade)
+#[no_mangle]
+pub unsafe extern "C" fn has_stable_actor() -> Value {
+    let metadata = PersistentMetadata::get();
+    if (*metadata).stable_actor.forward_if_possible() == DEFAULT_VALUE {
+        Value::from_scalar(0)
```

Comment:
Detail: Alternatively, one could also return a `bool`. I believe this would be more consistent to other RTS function exports, e.g. `contains_field`,

Reply:
Fixed